sql/item_weapon.sql

```diff
@@ -3556,10 +3556,15 @@ INSERT INTO `item_weapon` VALUES (20561,'carnwenhan',2,0,242,242,228,1,1,186,92,
 INSERT INTO `item_weapon` VALUES (20562,'carnwenhan',2,0,242,242,228,1,1,186,92,0);
 INSERT INTO `item_weapon` VALUES (20563,'twashtar',2,0,242,242,228,1,1,176,89,0);
 INSERT INTO `item_weapon` VALUES (20564,'twashtar',2,0,242,242,228,1,1,176,89,0);
+INSERT INTO `item_weapon` VALUES (20568,'wind_knife_+1',2,0,0,0,0,1,1,189,2,0);     -- Still needs added effect and enchantment
```

Comment:
```sql
-- Wind Knife +1
INSERT INTO `item_mods` VALUES (20568,431,1);   -- Additional effect: Wind Damage (Type DAMAGE)
INSERT INTO `item_mods` VALUES (20568,499,3);   -- Additional effect animation (subEffect)
INSERT INTO `item_mods` VALUES (20568,500,2);   -- Additional effect damage before resists
INSERT INTO `item_mods` VALUES (20568,501,10); -- Additional effect Chance 10%
INSERT INTO `item_mods` VALUES (20568,950,8);   -- Additional effect element Wind
```
That's how you add additional effect damage and the format I use.
3 is the subeffect value for wind damage animations, you can find these in the battleentity enum [here](https://github.com/LandSandBoat/server/blob/base/src/map/entities/battleentity.h#L291)

Proc rate of 10% is a guesstimate on my part. Most effects are 5 or 10 rarely an item like sirroco kukri is 100%

Damage should be the MOST the effect can hit for, resists will lower it. 

In nearly all cases for melee weapons NOTHING will increase the damage. Not your int, not your magic attack. Nothing. Ranged attacks DO use INT or MND (MND for light element dmg, INT for all other elements). In very rare cases items exist that use modifiers - we'll handle those if and when we hit them, so don't worry about those for now.


For the enchantment:
```lua
require("scripts/globals/spell_data")

player:castSpell(xi.magic.spell.AERO, target)
```
Could work, but I'm not sure if items targeting mobs is working.

Reply:
Sure, noted. For now though, I am trying to stick to completing the others first. There are still a few missing items, and many missing item_mods. And, well, you know me, it will take forever of asking and clarifying to get these right. So, for now, the comment I felt was just better so me or someone else later knows that it is not finished yet, and needs to be done, without taking a few weeks to try to get it done. (And yes, it would take me that long, as you know, I am not that great at those parts xD).
Later, I will make other submissions, and am trying to also figure out everything that is wrong/missing by going over each and every item (my previous commit). But, for this run, this is simply adding in some missing items and added some mods to some items that are there, but are incomplete (many lvl 99, ilvl 119 seem to be missing the item_weapon and item_mods).

Reply:
I will see what I can do on this. No guarantees though...

Reply:
Took the last option (per discussion on discord), and just left the whole script as a todo, other than headers and requirements.